pygmt/base_plotting.py

```diff
@@ -530,3 +534,71 @@ def image(self, imagefile, **kwargs):
         with Session() as lib:
             arg_str = " ".join([imagefile, build_arg_string(kwargs)])
             lib.call_module("image", arg_str)
+
+    @fmt_docstring
+    @use_alias(R="region", J="projection")
```

Comment:
```suggestion
    @use_alias(R="region", J="projection", D="position", F="box")
```
Create an alias for arguments F (box) and D (position), and update your docstring and tests to use these aliases instead.

Pinging @joa-quim. As per your comment at https://github.com/GenericMappingTools/pygmt/pull/329#issuecomment-536178749 on standardizing aliases, I noticed that GMT.jl uses `position` for [colorbar](https://github.com/GenericMappingTools/GMT.jl/blob/8cff1c46195534c87b95b8b5bfa4cba7e706bdb6/src/psscale.jl#L8) and [logo](https://github.com/GenericMappingTools/GMT.jl/blob/8cff1c46195534c87b95b8b5bfa4cba7e706bdb6/src/gmtlogo.jl#L13), but `ref_point` for [image](https://github.com/GenericMappingTools/GMT.jl/blob/8cff1c46195534c87b95b8b5bfa4cba7e706bdb6/src/psimage.jl#L12) and `refpoint` for [legend](https://github.com/GenericMappingTools/GMT.jl/blob/8cff1c46195534c87b95b8b5bfa4cba7e706bdb6/src/pslegend.jl#L18). Shall we standardize these to `position`?